pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -299,6 +299,12 @@ stdenv.mkDerivation {
       hardening_unsupported_flags+=" stackprotector fortify pie pic"
     ''
 
+    + optionalString targetPlatform.isWasm ''
+      hardening_unsupported_flags+=" stackprotector fortify pie pic"
```

Comment:
@ericson2314 do any of these hardening flags work?

Reply:
@matthewbauer Just like unwinding they all probably make no sense for WASM.